ibm/mas_devops/roles/suite_app_install/tasks/main.yml

```diff
@@ -15,17 +15,16 @@
 - name: Load variables
   include_vars: "vars/{{ mas_app_id }}.yml"
 
-# 2.1 Load default application spec, if none is provided by user
-- name: Load default application spec
-  when: mas_app_spec is not defined or mas_app_spec == ""
-  include_vars: "vars/defaultspecs/{{ mas_app_id }}.yml"
-
-# 2.2  Run Application Specific Tasks
+# 2.1  Run Application Specific Tasks
```

Comment:
What's the reason for re-ordering these?  Have you verified that neither of the application specific tasks require the values laoded in the vars file, becuase these vars will no longer be available when tasks/$appId.yml runs now?

Reply:
It was reordered because there the storage variable was overwritten by the second one at the time so the mas_app_spec became {} and resulted to using the default specs. @pseemung Maybe you could explain better.

Reply:
@andrercm assisted me because I could not get the storage classes to work properly.  The problem with the original order was that I defined the custom storage class for mvi after the` mas_app_spec` was already set.  The order was changed because it defines the default storage classes first, then you call the step that sets the mas_app_spec if that’s not defined.

If you believe this is not correct, please let me know how I would resolve this scenario.

Reply:
That's fine :+1: ... but my question remains unanswered:

> Have you verified that neither of the application specific tasks require the values laoded in the vars file

The focus of this PR is MVI, but IoT is also impacted by this change, has it been tested that nothing in the IoT app-specific task required those vars to have been set up before it ran?

Reply:
Oh I see. You want to make sure that IoT is working fine even with this change. @pseemung Do you have time today to test with IoT?